payments_config/__init__.py

```diff
@@ -69,7 +80,14 @@ def format_prices(self, locales):
         return prices
 
     def to_dump(self):
-        return self.__dict__
+        """
+        When dumping the product, dump the seller as well, but not the
+        products (removed in seller.to_dump), otherwise you'll be in a
+        recursive mess.
```

Comment:
this seems more like a comment than a docstring because it's not explaining the interface of ``to_dump()``.